include/aws/s3/s3_client.h

```diff
@@ -67,8 +67,8 @@ enum aws_s3_meta_request_tls_mode {
 };
 
 enum aws_s3_meta_request_compute_content_md5 {
-    AWS_MR_CONTENT_MD5_ENABLED,
     AWS_MR_CONTENT_MD5_DISABLED,
+    AWS_MR_CONTENT_MD5_ENABLED,
 };
```

Comment:
Not related, but maybe tweak the comments at line 119 as you mentioned https://github.com/awslabs/aws-crt-java/pull/337#pullrequestreview-691093035😉

Reply:
Fixed!